go/libraries/doltcore/sqle/cluster/controller.go

```diff
@@ -138,17 +138,18 @@ func NewController(lgr *logrus.Logger, cfg Config, pCfg config.ReadWriteConfig)
 
 	keyID := creds.PubKeyToKID(ret.pub)
 	keyIDStr := creds.B32CredsEncoding.EncodeToString(keyID)
+	audience := "dolthub-remote-api.liquidata.co" // TODO
```

Comment:
This is going to have to change to be a factory like `grpcCreds func(aud string) *creds.RPCCreds` or something --- we need to sign outbound requests differently depending on which replica we connect to.